frontend/src/components/router-links/TeamLink.vue

```diff
@@ -0,0 +1,38 @@
+<template>
+    <router-link
+        v-slot="{ href, navigate }"
+        v-bind="extendedProps"
+        custom
+    >
+        <a
+            v-bind="$attrs"
+            :href="href"
+            @click="navigate"
+        >
+            <slot />
+        </a>
+    </router-link>
+</template>
+
+<script>
+import { RouterLink as DefaultRouterLink } from 'vue-router'
+import { mapState } from 'vuex'
+
+export default {
+    name: 'TeamLink',
+    inheritAttrs: false,
+    props: {
+        ...DefaultRouterLink.props
+    },
+    computed: {
+        ...mapState('account', ['team']),
+        extendedProps () {
+            const props = { ...this.$props }
+            if (!props.to.params.team_slug) {
+                props.to.params.team_slug = this.team.slug
```

Comment:
and in a circumstance where `team` is `undefined`?

Reply:
that is a valid point, will have to go through the sign-up procedure and check but from the top of my head users shouldn't encounter team-links only after being assigned to a team and have the full ui at their disposal.

I'll have a look over it later after this current release when i'll have some spare time on my hands, this is not an important task

Reply:
@cstns reminder for an action here too